### PR TITLE
Consume new docker images

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary
Test new docker images in ci

### Additional Details

@cerner/terra

[CONTRIBUTORS.md]: https://github.com/cerner/terra-dev-site/blob/master/CONTRIBUTORS.md
[License]: https://github.com/cerner/terra-dev-site/blob/master/License.md

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,21 +1,6 @@
 version: '3'
 
 services:
-  dev: &dev
```
Comment (emilyrohrbough) [inline docker-compose.yml]: can we keep the dev service but use the `term` service setup?

File: Dockerfile [hunk 2]
```diff
@@ -1,2 +1 @@
-ARG project_name=terra-dev-site
-FROM cerner/terra-node
+FROM mjhenkes/terra-node-ci
```
Comment (emilyrohrbough) [inline Dockerfile]: I know we are swapping the base image out for alpine, but should our consumers feel this affect or care? Can we bump the terra-node version to v2 to use alpine and anyone pulling latest can get these changes without changing the image?

also I assume terra is the only one using this 😄

File: docker-compose.yml (hunk 1 above)
Comment (mjhenkes) [reply docker-compose.yml]: you just want to call it dev instead of term?

File: Dockerfile (hunk 2 above)
Comment (mjhenkes) [reply Dockerfile]: Consumers don't care, we just never versioned terra-node and I didn't want to have to go and fix all our repos at once. All the other repos are just pulling in terra-node latest and would have all broken.

File: docker-compose.yml (hunk 1 above)
Comment (emilyrohrbough) [reply docker-compose.yml]: Yeah I think dev makes more sense. We can just stay zshell is our default dev terminal

**Comment by emilyrohrbough** [comment]
+1 once the toolkit images are used

**Pull request merged**